playbooks/gce/openshift-cluster/launch.yml

```diff
@@ -44,17 +44,4 @@
   - add_host: name={{ a_infra }} groups=service_master
```

Comment:
Yes, it appears to be used here:

https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-master/service.yml#L14